02Lesson/01Homework/index.html

```diff
@@ -1,8 +1,32 @@
 <!DOCTYPE html>
 <html>
 <head>
-  <title></title>
+  <meta charset="utf-8">
+  <title>Savage Bio</title>
 </head>
 <body>
+  <img style="height:20%; width: 20%" src = "https://scontent-mia1-1.xx.fbcdn.net/v/t1.0-9/10653714_10204131189073222_5438750846180900030_n.jpg?oh=78f887afeb78a1dfb85465779e5484f9&oe=58A3F75D" alt="Russ savage"/>
+  <h1>Russ Savage</h1>
+  <hr>
+  <p>
+  I moved to Austin 4 years ago and love it here. The summers can be brutal but after living in extremely cold places like South Dakota, I can tolerate heat in the summer in order to have 50 degree days in the winter.
+  </p>
+  <h2>Hobbies and Interests</h2>
+  <p>My primary hobby right now is racing RC cars. I race both on and off road. I run 2 classes in both types of racing</p>
+  <img style="height:20%; width: 20%" src="https://scontent-mia1-1.xx.fbcdn.net/v/t1.0-9/13439232_10208523386995425_2067274129786750026_n.jpg?oh=a8b35f8e7d963d4417b694db5059c79e&oe=585FCFCB" alt="RC Cars">
+    <ul>
+      <li><h3>On Road Classes</h3>
```

Comment:
I would put tags of block elements on their own line:
```
<li>
  <h3>
     On Road Classes
  </h3>
  <ol>
     ...
  </ol>
</li>
```
That way you see immediately that indentation for `<ol>` tag is on the wrong level which leads to the missing closing ´</li>` tag a few lines further down.